run_evaluation.py

```diff
@@ -9,21 +9,21 @@
 # %matplotlib inline
```

Comment:
should we move these `run_evaluation` and `run_visualization` into an outdated folder?

Reply:
yes I'll do that now